### PR TITLE
Update santactl fileinfo, sync, and status to show teamID info

### PR DESCRIPTION
**Pull request by tnek**
Some more QOL for working and managing team ID rules. Let me know if there's anywhere else I'm missing

Linked issue: https://github.com/google/santa/issues/634

### CODE REVIEW

File: docs/details/rules.md [hunk 1]
```diff
@@ -64,14 +64,23 @@ chain's intermediates or roots has no effect on binaries signing by a leaf.
 Santa ignores the chain and is only concerned with the leaf certificate's
 SHA-256 hash.
 
+##### Apple Developer Team ID Rules
+The Apple Developer Program Team ID is a 10-character identifier  issued by Apple
```
Comment (russellhancox) [inline docs/details/rules.md:68]: double space after identifier

File: docs/details/rules.md [hunk 2]
```diff
@@ -64,14 +64,23 @@ chain's intermediates or roots has no effect on binaries signing by a leaf.
...
+and tied to developer accounts/organizations. This is distinct from Certificates,
+as a single developer account can and frequently will request/rotate between 
+multiple different signing certificates and entitlements.
+
+This is an even more powerful rule with broader reach than individual certificate rules.
+
+
```
Comment (russellhancox) [inline docs/details/rules.md:75]: Remove extra line

**Review by russellhancox — APPROVED** [review]

File: docs/details/rules.md (hunk 2 above)
Comment (tnek) [reply docs/details/rules.md:73]: Done

File: docs/details/rules.md (hunk 1 above)
Comment (tnek) [reply docs/details/rules.md]: Done

**Review by russellhancox — APPROVED** [review]

**Pull request merged**